src/Edit.js

```diff
@@ -64,6 +64,13 @@ import Controls from './components/Controls';
  */
 import Inspector from './components/Inspector';
 
+import useGetMediaType from './hooks/useGetMediaType';
```

Comment:
Would you please import the hook from `wp-react-hooks` 🙂?

Reply:
Oh. I have just removed the line 🙂 Thank you 👍